.evergreen.yml

```diff
@@ -47,6 +47,20 @@ functions:
     params:
       files:
         - "./parsley/bin/jest/*.xml"
+
+  attach-storybook:
+    command: s3.put
+    type: system
+    params:
+      aws_key: ${aws_key}
+      aws_secret: ${aws_secret}
+      local_files_include_filter:
+        ["parsley/storybook-static/*"]
+      remote_file: parsley/${task_id}/storybook/
+      bucket: mciuploads
+      content_type: text/html
+      permissions: public-read
+      display_name: storybook_
```

Comment:
Hmm, the ultimate goal of these files is that clicking on `<display_name>index.html` in the Files tab should link to a working Storybook build, but clicking `storybook_index.html` here points to a broken site. Is it possible that using `storybook-` here like in Spruce would fix it?